tests/benchdnn/dnn_types.cpp

```diff
@@ -190,6 +192,7 @@ int attr_t::get_default_mask(policy_t policy, int ndims) {
             assert(ndims > 0 && ndims <= DNNL_MAX_NDIMS);
             return (1 << ndims) - 1;
         case COMMON: return 0;
+        case HOST_SCALAR: return 0;
```

Comment:
It feels a little bit ~~dangerous~~ discomforting to make this returning `0`. The API doesn't rely on the mask, neither should rest part of benchdnn, so I think it's better to return `INT_MIN >> 1` and see what fails. It shouldn't be dramatic if at all.

Reply:
I'm not sure I agree, could you elaborate why you think `HOST_SCALAR` should behave differently?

API for setting host scale doesn't require mask because it is redundant since applying a single value to the whole tensor is operation with mask == 0 always, so it would be confusing to have mask in API.
The result of applying host scale is the same as applying "regular" scale with mask == 0 except for the host one requiring a simple scalar value residing on host.